net-libs/libnice/patches/libnice_x86-0.1.13.patchset

```diff
@@ -0,0 +1,93 @@
+From d5f26c32ad9d2c2a3f345504f58244f9bc110330 Mon Sep 17 00:00:00 2001
+From: Steven Kou <kouyongkang@gmail.com>
+Date: Sat, 26 Dec 2015 23:18:02 +0000
+Subject: Search for socket under network instead of inet
+
+
+diff --git a/configure.ac b/configure.ac
+index 223b883..f496ec0 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -167,7 +167,7 @@ AS_IF([test "x$enable_compile_warnings" = "xerror"],[
+ # Fixes for Solaris
+ #
+ AC_SEARCH_LIBS([inet_pton],[nsl])
+-AC_SEARCH_LIBS([socket],[socket inet])
++AC_SEARCH_LIBS([socket],[socket network])
+ case $host in
+ 	*-*-solaris* )
+ 	AC_DEFINE(_XOPEN_SOURCE,          600, Needed to get declarations for msg_control and msg_controllen on Solaris)
+-- 
+2.2.2
+
+
+From 7454b2e84cecf17fccff24664c7f4221c5ccab82 Mon Sep 17 00:00:00 2001
+From: Steven Kou <kouyongkang@gmail.com>
+Date: Sat, 26 Dec 2015 23:21:00 +0000
+Subject: Make interfaces.c compile under Haiku.
+
+
+diff --git a/agent/interfaces.c b/agent/interfaces.c
+index 0fa2fd7..b60e925 100644
+--- a/agent/interfaces.c
++++ b/agent/interfaces.c
+@@ -46,7 +46,7 @@
+ #include "interfaces.h"
+ #include "agent-priv.h"
+ 
+-#ifdef G_OS_UNIX
++#ifdef G_OS_HAIKU
```

Comment:
This is not the right way to do this. If we ever wanted to upstream this patch it would be rejected. You must not break the build for other systems. Change this (and other similar changes) to `#if defined(G_OS_UNIX) || defined(G_OS_HAIKU)`.